openmetadata-service/src/main/java/org/openmetadata/service/security/auth/LoginAttemptCache.java

```diff
@@ -0,0 +1,57 @@
+package org.openmetadata.service.security.auth;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+public class LoginAttemptCache {
+  private final int MAX_ATTEMPT = 3;
+  private final LoadingCache<String, Integer> attemptsCache;
+
+  public LoginAttemptCache() {
+    super();
+    attemptsCache =
+        CacheBuilder.newBuilder()
+            .maximumSize(1000)
+            .expireAfterWrite(10, TimeUnit.MINUTES)
```

Comment:
Can we make this 10 minutes configurable?

Reply:
Once we make it configurable, we can write unit tests around it by reducing this time to probably 30 seconds in the tests.

Reply:
@vivekratnavel at constructor level or at openmetdata.yml?

Reply:
done